python.cmake

```diff
@@ -77,25 +77,20 @@ GET_FILENAME_COMPONENT(PYTHON_LIBRARY_DIRS "${PYTHON_LIBRARIES}" PATH)
 MESSAGE(STATUS "PythonLibraryDirs: ${PYTHON_LIBRARY_DIRS}")
 MESSAGE(STATUS "PythonLibVersionString: ${PYTHONLIBS_VERSION_STRING}")
 
-# Default Python packages directory
-SET(PYTHON_PACKAGES_DIR site-packages)
-
 # Use either site-packages (default) or dist-packages (Debian packages) directory
 OPTION(PYTHON_DEB_LAYOUT "Enable Debian-style Python package layout" OFF)
 
-IF (PYTHON_DEB_LAYOUT)
-  SET(PYTHON_PACKAGES_DIR dist-packages)
-ENDIF (PYTHON_DEB_LAYOUT)
-
 EXECUTE_PROCESS(
   COMMAND "${PYTHON_EXECUTABLE}" "-c"
-  "import sys, os; print(os.sep.join(['lib', 'python' + sys.version[:3], '${PYTHON_PACKAGES_DIR}']))"
+  "from distutils import sysconfig; print(sysconfig.get_python_lib(prefix='', plat_specific=True))"
   OUTPUT_VARIABLE PYTHON_SITELIB
+  OUTPUT_STRIP_TRAILING_WHITESPACE
   ERROR_QUIET)
 
-# Remove final \n of the variable PYTHON_SITELIB
-STRING(REPLACE "\n" "" PYTHON_SITELIB "${PYTHON_SITELIB}")
-NORMALIZE_PATH(PYTHON_SITELIB)
+# Keep compatility with former jrl-cmake-modules versions
+IF(PYTHON_DEB_LAYOUT)
```

Comment:
With the above command, is this option still required or doing anything? On Debian, the above command should give you `dist-packages` already (or is this for cross-compiling?)

Reply:
I know that people from JRL are still dependent on this variable. I prefer to keep compatibility with them. But I do agree, this is no more needed.